junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/Arguments.java

```diff
@@ -125,6 +129,80 @@ static ArgumentSet argumentSet(String name, @Nullable Object... arguments) {
 		return new ArgumentSet(name, arguments);
 	}
 
+	/**
+	 * Factory method for creating an instance of {@code Arguments} based on
+	 * the supplied {@link List} of {@code arguments}.
+	 *
+	 * @param arguments the arguments to be used for an invocation of the test
+	 * method; must not be {@code null} but may contain {@code null}.
+	 * @return an instance of {@code Arguments}; never {@code null}.
+	 * @since 6.0
```

Comment:
```suggestion
	 * @param arguments the arguments to be used for an invocation of the test
	 * method; must not be {@code null} but may contain {@code null}
	 * @return an instance of {@code Arguments}; never {@code null}
	 * @since 6.0
```